src/eduid_webapp/eidas/acs_actions.py

```diff
@@ -236,10 +230,11 @@ def nin_verify_BACKDOOR(user: User) -> Response:
     try:
         nin_element = NinProofingElement.from_dict(dict(number=asserted_nin, created_by='eduid-eidas', verified=False))
         proofing_state = NinProofingState(id=None, modified_ts=None, eppn=user.eppn, nin=nin_element)
-        verify_nin_for_user(user, proofing_state, proofing_log_entry)
-    except AmTaskFailed as e:
-        current_app.logger.error('Verifying NIN for user failed')
-        current_app.logger.error('{}'.format(e))
+        if not verify_nin_for_user(user, proofing_state, proofing_log_entry):
+            current_app.logger.error(f'Failed verifying NIN for user {user}')
+            return redirect_with_msg(redirect_url, ':ERROR:Temporary technical problems')
```

Comment:
Shouldnt this also be "return redirect_with_msg(redirect_url, CommonMsg.temp_problem)"?

Reply:
I'm all for changing it, but the string is what is used in current master:

https://github.com/SUNET/eduid-webapp/blob/ab99bca053dd49fe151336a455035d457368f1a7/src/eduid_webapp/eidas/acs_actions.py#L243